src/lightning_lite/utilities/distributed.py

```diff
@@ -0,0 +1,264 @@
+import logging
```

Comment:
Some functions from this file need to remain in lightning, which is why this is not a clean move.

Reply:
No implementation changes to the moved functions here.